src/main/java/ua/ukr/net/model/Department.java

```diff
@@ -0,0 +1,55 @@
+package ua.ukr.net.model;
+
+import java.util.List;
+
+public class Department {
+    private String name;
+    List<Employee> employeeDepartment;
```

Comment:
field employeeDepartment should also be private, and it is better to rename it to employees